README.Rmd

```diff
@@ -194,7 +194,7 @@ to users to run canned examples (or analyses). They can be accessed once
     + [dplyr](https://dplyr.tidyverse.org) and [tidyr](https://tidyr.tidyverse.org) verb
       S3 methods for the `soma_adat` class.
     + `?rownames` helpers that do not break `soma_adat` attributes.
-    + please see vignette `vignette("tips-loading-and-wrangling", package = "SomaDataIO")`
+    + please see vignette [Loading and Wrangling 'SomaScan'](https://somalogic.github.io/SomaDataIO/articles/tips-loading-and-wrangling.html)
```

Comment:
I think, for the sake of clarity, we might want to refer to these as "articles" and "workflows" (when appropriate) to reduce confusion for the reader, since that is how they're listed on the pkgdown site. So this could be modified to say "please see the article Loading and Wrangling...."